test/y2storage/proposal/autoinst_devices_planner_test.rb

```diff
@@ -187,6 +191,26 @@
         expect(root.filesystem_type).to eq(Y2Storage::Filesystems::Type::EXT4)
         expect(home.filesystem_type).to eq(Y2Storage::Filesystems::Type::XFS)
       end
+
+      it "sets the mountby properties" do
```

Comment:
Here the filesystem is checked, but not the mountby, right?

Reply:
Yes, you are absolutely right. I will fix it.